setup_docker.sh

```diff
@@ -1,5 +1,5 @@
 #kills the running process, and creates a new web-server to run
 docker kill "$(docker ps -q)"
 docker load -i webserver_img
-docker run -d -p 80:80 httpserver ./web-server simple_config
+docker run -d -p 80:2020 httpserver ./web-server simple_config
```

Comment:
because we need to connect to the aws server over port 80 so I dont need to say <aws-url>:2020/, but rather I can just type http://<aws-url>/ into google chrome and it will work.